backend/BackendMIIT/src/main/java/com/example/BackendMIIT/service/DirectionService.java

```diff
@@ -0,0 +1,12 @@
+package com.example.BackendMIIT.service;
+
+import org.jsoup.nodes.Element;
+
+import java.util.List;
+
+public interface DirectionService {
+
+    void parseDirections(String url);
+
```

Comment:
Extra empty line. We separate different "sentences" in the code with an empty line to improve readibility. In interfaces between methods there is no need to take empty lines if the interface methods are written in one line

Reply:
There's IntelliJ's refactor feature

Reply:
Ok